ui/app/common/patient-search/controllers/patientsListController.js

```diff
@@ -75,19 +86,59 @@ angular.module('bahmni.common.patientSearch')
             $(container).children('patient-list-spinner').hide();
         };
 
-        $scope.getHeadings = function (patients) {
-            if (patients && patients.length > 0) {
-                var headings = _.chain(patients[0])
+        $scope.getHeadings = function () {
+            if ($scope.search.activePatients && $scope.search.activePatients.length > 0) {
+                var headings = _.chain($scope.search.activePatients[0])
                     .keys()
                     .filter(function (heading) {
                         return _.indexOf(Bahmni.Common.PatientSearch.Constants.tabularViewIgnoreHeadingsList, heading) === -1;
```

Comment:
I believe it would be beneficial to keep `tabularViewIgnoreHeadingsList` configurable instead, allowing implementers to have full control over displayed content when modifying emrapi.sqlSearch.activePatientsByLocation.

Reply:
Same as above